README.md

```diff
@@ -122,7 +122,13 @@ Also, we also provide Optuna docker images on [DockerHub](https://hub.docker.com
 
 ## Contribution
 
-Any contributions to Optuna are welcome! When you send a pull request, please follow the
+Any contributions to Optuna are welcome!
+
+If you are a new contributor, please check [good first issues](https://github.com/optuna/optuna/labels/good%20first%20issue). These are simple and well-defined so that they can be good starting points for you to get familiar with the contribution workflow and other developers.
+
+If you have already contributed to Optuna, I would highly recommend the other [contribution-welcome issues](https://github.com/optuna/optuna/labels/contribution-welcome).
```

Comment:
Sorry, again details.
```suggestion
If you already have contributed to Optuna, we recommend the other [contribution-welcome issues](https://github.com/optuna/optuna/labels/contribution-welcome).
```

Reply:
Thank you for your suggestion. I applied the latter half in https://github.com/optuna/optuna/pull/1597/commits/bf427954918db5c37a01f374fb1f3413a4a0efdb.
I'm not pretty sure of the order of `already` since we can see `have already contributed to` in the [scikit-learn's document](https://scikit-learn.org/dev/developers/contributing.html#issues-for-new-contributors). So, let me keep the current change for now.

Reply:
To emphasize `already`, I applied your suggestion in https://github.com/optuna/optuna/pull/1597/commits/1872cddef30847f994d40d6e6097119344fe71f8.